### ISSUE TITLE
500 on PATCH

### ISSUE THREAD

**Issue by michielbdejong** [michielbdejong/wac-ldp#3]
```
curl 'https://lolcathost.de/storage/michiel5/profile/card' -X PATCH -H 'Connection: keep-alive' -H 'Pragma: no-cache' -H 'Cache-Control: no-cache' -H 'authorization: Bearer eyJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJTVHpKVXNkakdod2tyNzFFSFZFaWsiLCJhdWQiOiJodHRwczovL2xvbGNhdGhvc3QuZGUiLCJleHAiOjE1ODU3NzIxMjMsImlhdCI6MTU4NTc2ODUyMywiaWRfdG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSXNJbXRwWkNJNkluaGxUMnBsY3psMU0wRmpWVFJNUW5wallXNUZUVGR3V2t4M1UyeDRZVTQzVlRZeVducFBRa1JSZFhjaWZRLmV5SnpkV0lpT2lKb2RIUndjem92TDJ4dmJHTmhkR2h2YzNRdVpHVXZjM1J2Y21GblpTOXRhV05vYVdWc05TOXdjbTltYVd4bEwyTmhjbVFqYldVaUxDSnViMjVqWlNJNklqQnNPWGRKYlRCcFRUZzVORXc0ZDBkaGIyRkVXbWRXVkVOa2FtWXhRVGt4WHpOdkxWUnRibFoyZFVraUxDSmhkRjlvWVhOb0lqb2lWRVJVZEVOa1dWQjNhRkJuZDFkNllWQklYMDUzUVNJc0luTmZhR0Z6YUNJNklsTktTVTg1WTFaU1VEVjFSMlowVlZWWFUwbzNRa0VpTENKamJtWWlPbnNpWVd4bklqb2lVbE15TlRZaUxDSmxJam9pUVZGQlFpSXNJbVY0ZENJNmRISjFaU3dpYTJWNVgyOXdjeUk2V3lKMlpYSnBabmtpWFN3aWEzUjVJam9pVWxOQklpd2liaUk2SWpWZlRHWnBkSHBHTldWV1IzWmtRaTFWZUROUmJqVmZlVVZPWVdOWlNEbHpORGh2YzFSUVYzSTNRMWxEWTJoc1VraGZjbU5LYkhSMVJuQkJlRlpRV2xKRk5GUXhhWEZCYlMwMFQycHZXbmRhUVRCaU5UTllNbFZzY3psME4yTlpTR0ZYVEV3MFdXcFJURGc1U1dOaVkycGhVM1F3YjJwNFRWbDJOWFpuUlVsdk1VeEVTMFYxUTBaNFduVldSSEExY21oemVIVXdaalprT1RWM1lqTmlTbEpRUWpoaWRrZE1RVE52WjFOM2RGaExNMmxmTmpNMlIzWm5SbE14ZFdGR1RVODFTRlozY25KQ2FYSjVXbEpLWW0xV1NVdGtUVTkxZW05NWNVRjZRWE56VUVacUxVNW9ORGswYUcxVldsQm5PRkJVWkhsc09YWTBVbWR1UWxsdGNFRlZiVjgyWjFaa1RuVmpZVFJCWVZwQk0xZFpaSFF6VmtWallXVlNYMm80Y25jeVVucE9kUzFaYWtwRFNESXlVbGhVUW1wcVh6aG9iazV3VFdSRlFXRkZjVVl5ZEZwTlkxbGZTemhCY2toeU1UTklaSGxyZHlKOUxDSmhkV1FpT2lKVFZIcEtWWE5rYWtkb2QydHlOekZGU0ZaRmFXc2lMQ0psZUhBaU9qRTFPRFUzTnpFeE1UTXNJbWxoZENJNk1UVTROVGMyTnpVeE15d2lhWE56SWpvaWFIUjBjSE02THk5c2IyeGpZWFJvYjNOMExtUmxJbjAuRkdsSi1nVzBJUHZNd3pFNDJMZUNfYk9Ncnh3RVNqaTI2VktjdTVMZ2RHMm82Z1ZNY3dmZTRMYlpxWVd3cUl4Z1NvU05wazVXNFcyNU5TVkV0RmdJTnRUUnlQcGZ5UjZSQ3N4XzJDd0ljd2thQTBnQUZsUjJ2LVBBblAzcFJwMk05MU9PQmtIRUNlTURwOGJiU1pEQTFFY0xrck5tR3VuZG1IUURpcUdva3luYTJSSk1uX3J4X2FxbV8yV1JfUHRRNDFJMHB5RUZVOFlTZ0dTQkRBd3NRVDQ5NHdHT0trRDBGZWREalVVUGF4UUp1cFdUVDhWbjR3TnNHTVRjQVZuMzM0clhlV2dXUlljeDZ0OFBuYUxCOF9OTEFkTVJWYjFISmNrdFZTeUIzaWM3bS15elRXWlBmSlZpSVl6NzdMTzJaS0hhZUh4Z1RLU25yazZzUW9QUWdBIiwidG9rZW5fdHlwZSI6InBvcCJ9.Fa50qSaYZ0DMgiBsYrCdN5894lGugKO6f3NmvVLgn8EKPYhfCzNNUSh6E63CW1nLoJl3VpgWIi8F1Iok86MQGQDSxo0T3lyxWyCH7jhpbUTxjz9dcarR9QXgpGMSacvl1_L-fybOwzZ66-6PUkbtyH9DlaUm3RjiMZpTpbj9OyL7OFiIBMIVgMaM1FN1RRSwmyPqNNCx3hPVQeVfPStHYqBknW8meALSOQh5sQevsySN8qmgQF4okbu67-bmnummI-J-_YVaIbh5m-TaAhZP6G60skFYu9m5UDbFcKCRPZuFU-tT9_ho-blDWoGiVkq0Nkc1Thrf078JvPnSalkWsQ' -H 'Sec-Fetch-Dest: empty' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.162 Safari/537.36' -H 'Content-Type: application/sparql-update' -H 'Accept: */*' -H 'Origin: https://lolcathost.de' -H 'Sec-Fetch-Site: same-origin' -H 'Sec-Fetch-Mode: cors' -H 'Referer: https://lolcathost.de/profile/https%3A%2F%2Fsharonstrats.inrupt.net%2Fprofile%2Fcard%23me' -H 'Accept-Language: nl-NL,nl;q=0.9,en-US;q=0.8,en;q=0.7,fr-FR;q=0.6,fr;q=0.5,de-DE;q=0.4,de;q=0.3,es-ES;q=0.2,es;q=0.1,id-ID;q=0.1,id;q=0.1' -H 'Cookie: _session=ynhqxmuCo3TGynAL2m5Th' --data-binary $' INSERT DATA {<https://lolcathost.de/storage/michiel5/profile/card#me> <https://www.w3.org/TR/activitypub/#following> <https://lolcathost.de/storage/michiel5/friends-eb3563b2-8329-450d-91d8-5bfad56b8c2d#this>.\n<https://lolcathost.de/storage/michiel5/friends-eb3563b2-8329-450d-91d8-5bfad56b8c2d#this> <http://www.w3.org/ns/ldp#inbox> <https://lolcathost.de/storage/michiel5/friend-requests-inbox/>.\n}' --compressed
```

```
  update-blob-handler operation updateBlob! { appendOnly: false } +14s
  Caching RDF Layer before patch {<https://lolcathost.de/storage/michiel5/profile/card#me> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Person> .
<https://lolcathost.de/storage/michiel5/profile/card#me> <http://www.w3.org/ns/pim/space#storage> <https://lolcathost.de/storage/michiel5/> .
<https://lolcathost.de/storage/michiel5/profile/card#me> <http://www.w3.org/ns/ldp#inbox> <https://lolcathost.de/storage/michiel5/inbox/> .
<https://lolcathost.de/storage/michiel5/profile/card#me> <http://www.w3.org/ns/auth/acl#trustedApp> <https://lolcathost.de/storage/michiel5/profile/card#same-origin> .
<https://lolcathost.de/storage/michiel5/profile/card#same-origin> <http://www.w3.org/ns/auth/acl#origin> <https://lolcathost.de> .
<https://lolcathost.de/storage/michiel5/profile/card#same-origin> <http://www.w3.org/ns/auth/acl#mode> <http://www.w3.org/ns/auth/acl#Read> .
<https://lolcathost.de/storage/michiel5/profile/card#same-origin> <http://www.w3.org/ns/auth/acl#mode> <http://www.w3.org/ns/auth/acl#Write> .
<https://lolcathost.de/storage/michiel5/profile/card#same-origin> <http://www.w3.org/ns/auth/acl#mode> <http://www.w3.org/ns/auth/acl#Control> .} +14s
  app errored Error: Object {<https://lolcathost.de/storage/michiel5/profile/card#me> <https://www.w3.org/TR/activitypub/#following> <https://lolcathost.de/storage/michiel5/friends-eb3563b2-8329-450d-91d8-5bfad56b8c2d#this> .
<https://lolcathost.de/storage/michiel5/friends-eb3563b2-8329-450d-91d8-5bfad56b8c2d#this> <http://www.w3.org/ns/ldp#inbox> <https://lolcathost.de/storage/michiel5/friend-requests-inbox/> .} is not an object type
    at IndexedFormula.add (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/rdflib/lib/store.js:441:15)
    at sparqlUpdateParser (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/rdflib/lib/patch-parser.js:93:12)
    at CachingRdfLayer.<anonymous> (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:54:33)
    at Generator.next (<anonymous>)
    at /Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:4:12)
    at CachingRdfLayer.applyPatch (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:46:16)
    at Object.<anonymous> (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/operationHandlers/updateBlobHandler.js:30:46)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/operationHandlers/updateBlobHandler.js:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:85:5) +6ms

**Comment by michielbdejong** [comment]
Looks like
https://github.com/michielbdejong/wac-ldp/blob/master/src/lib/rdf/CachingRdfLayer.ts#L40
calls https://github.com/linkeddata/rdflib.js/blob/master/src/patch-parser.js
and ends up triggering
https://github.com/linkeddata/rdflib.js/blob/master/src/store.ts#L433

Should write a unit test to reproduce this!

**Comment by michielbdejong** [comment]
Progress! But still:
```sh
 update-blob-handler operation updateBlob! { appendOnly: false } +27s
  Caching RDF Layer before patch {<https://lolcathost.de/storage/michiel5/profile/card#me> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Person> .
<https://lolcathost.de/storage/michiel5/profile/card#me> <http://www.w3.org/ns/pim/space#storage> <https://lolcathost.de/storage/michiel5/> .
<https://lolcathost.de/storage/michiel5/profile/card#me> <http://www.w3.org/ns/ldp#inbox> <https://lolcathost.de/storage/michiel5/inbox/> .
<https://lolcathost.de/storage/michiel5/profile/card#me> <http://www.w3.org/ns/auth/acl#trustedApp> <https://lolcathost.de/storage/michiel5/profile/card#same-origin> .
<https://lolcathost.de/storage/michiel5/profile/card#same-origin> <http://www.w3.org/ns/auth/acl#origin> <https://lolcathost.de> .
<https://lolcathost.de/storage/michiel5/profile/card#same-origin> <http://www.w3.org/ns/auth/acl#mode> <http://www.w3.org/ns/auth/acl#Read> .
<https://lolcathost.de/storage/michiel5/profile/card#same-origin> <http://www.w3.org/ns/auth/acl#mode> <http://www.w3.org/ns/auth/acl#Write> .
<https://lolcathost.de/storage/michiel5/profile/card#same-origin> <http://www.w3.org/ns/auth/acl#mode> <http://www.w3.org/ns/auth/acl#Control> .} +27s
  Caching RDF Layer patchObject {
  query: NamedNode {
    termType: 'NamedNode',
    value: 'https://lolcathost.de/storage/michiel5/profile/card#query'
  },
  insert: IndexedFormula {
    termType: 'Graph',
    statements: [ [Statement], [Statement] ],
    constraints: [],
    initBindings: [],
    optional: [],
    propertyActions: [
      '<http://www.w3.org/1999/02/22-rdf-syntax-ns#type>': [Array],
      '<http://www.w3.org/2002/07/owl#sameAs>': [Array]
    ],
    classActions: [
      '<http://www.w3.org/2002/07/owl#InverseFunctionalProperty>': [Array],
      '<http://www.w3.org/2002/07/owl#FunctionalProperty>': [Array]
    ],
    redirections: [],
    aliases: [],
    HTTPRedirects: [],
    subjectIndex: [
      '<https://lolcathost.de/storage/michiel5/profile/card#me>': [Array],
      '<https://lolcathost.de/storage/michiel5/friends-fa0ed2d5-ac55-4990-8ab4-af376eee7ffc#this>': [Array]
    ],
    predicateIndex: [
      '<https://www.w3.org/TR/activitypub/#following>': [Array],
      '<http://www.w3.org/ns/ldp#inbox>': [Array]
    ],
    objectIndex: [
      '<https://lolcathost.de/storage/michiel5/friends-fa0ed2d5-ac55-4990-8ab4-af376eee7ffc#this>': [Array],
      '<https://lolcathost.de/storage/michiel5/friend-requests-inbox/>': [Array]
    ],
    whyIndex: [
      '<https://lolcathost.de/storage/michiel5/profile/card>': [Array]
    ],
    index: [ [Array], [Array], [Array], [Array] ],
    namespaces: {},
    features: [ 'sameAs', 'InverseFunctionalProperty', 'FunctionalProperty' ]
  }
} +0ms
  app errored TypeError: iri.includes is not a function
    at new NamedNode (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/rdflib/lib/named-node.js:49:14)
    at IndexedFormula.sym (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/rdflib/lib/formula.js:683:14)
    at /Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:61:105
    at new Promise (<anonymous>)
    at CachingRdfLayer.<anonymous> (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:60:19)
    at Generator.next (<anonymous>)
    at /Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:4:12)
    at CachingRdfLayer.applyPatch (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:46:16)
    at Object.<anonymous> (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/operationHandlers/updateBlobHandler.js:30:46)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/operationHandlers/updateBlobHandler.js:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:85:5) +5ms
  app error has no resultType iri.includes is not a function TypeError: iri.includes is not a function
    at new NamedNode (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/rdflib/lib/named-node.js:49:14)
    at IndexedFormula.sym (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/rdflib/lib/formula.js:683:14)
    at /Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:61:105
    at new Promise (<anonymous>)
    at CachingRdfLayer.<anonymous> (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:60:19)
    at Generator.next (<anonymous>)
    at /Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:4:12)
    at CachingRdfLayer.applyPatch (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/rdf/CachingRdfLayer.js:46:16)
    at Object.<anonymous> (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/operationHandlers/updateBlobHandler.js:30:46)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/operationHandlers/updateBlobHandler.js:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:85:5) +0ms
  app response is ErrorResult: error result
    at WacLdp.<anonymous> (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/core/WacLdp.js:129:32)
    at Generator.throw (<anonymous>)
    at rejected (/Users/michiel/gh/michielbdejong/solid-app-kit/node_modules/wac-ldp/dist/lib/core/WacLdp.js:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:85:5) {
  resultType: 11
} +0ms
```
from
```
curl 'https://lolcathost.de/storage/michiel5/profile/card' -X PATCH -H 'Connection: keep-alive' -H 'Pragma: no-cache' -H 'Cache-Control: no-cache' -H 'authorization: Bearer eyJhbGciOiJSUzI1NiJ9.eyJpc3MiOiI2alNyQ1M5cmtLWktHRnRwMENVVDIiLCJhdWQiOiJodHRwczovL2xvbGNhdGhvc3QuZGUiLCJleHAiOjE1ODU5MTAzMTMsImlhdCI6MTU4NTkwNjcxMywiaWRfdG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSXNJbXRwWkNJNkluaGxUMnBsY3psMU0wRmpWVFJNUW5wallXNUZUVGR3V2t4M1UyeDRZVTQzVlRZeVducFBRa1JSZFhjaWZRLmV5SnpkV0lpT2lKb2RIUndjem92TDJ4dmJHTmhkR2h2YzNRdVpHVXZjM1J2Y21GblpTOXRhV05vYVdWc05TOXdjbTltYVd4bEwyTmhjbVFqYldVaUxDSnViMjVqWlNJNklsOURiR05aYnpCdmFGSk9VWEpFVldGTE9Fa3pYelJIUTB0VU1pMWphakpzTmpjMUxXdExlbEZUYW04aUxDSmhkRjlvWVhOb0lqb2lWbWs0TkZSWlNFUmpiMk5LYVdaeVQzRnJhMTh3UVNJc0luTmZhR0Z6YUNJNklrOUdjR1EzYjJNMmIyeFpkemR0UlVoZlMwVnFVRUVpTENKamJtWWlPbnNpWVd4bklqb2lVbE15TlRZaUxDSmxJam9pUVZGQlFpSXNJbVY0ZENJNmRISjFaU3dpYTJWNVgyOXdjeUk2V3lKMlpYSnBabmtpWFN3aWEzUjVJam9pVWxOQklpd2liaUk2SW5SaU56VkNWbGt4TFdKUFdsTXRWV3MxUVRoWlRrRkVTazExYmpKUFZtdGlkbFptVlRONmNsRlRjRWRJVDNNeWN6aENTVzQyT0hCaFptOVJiMHRhTms0M1dETjJXbkYyYVMxMFZIRnpURWRWY1hWYVNGSlFZVzFPVmxOSGRXVm9hWFI2Wmt4d0xYZEVaRVpWZWxkbWJqTTBVMVJsUlVaQ1FubEZXRGxCVUZBeWIxVk1hMEpGUlZnMlltSnVkbHBGWkhZMVozQnlPRmcyWDAxYWFHOWhlVWgzTTBKak5IaHdTRGQyYmtkVVRHWkRXVk5ETVhSNFpsZHdlVU4xV2xoU1lXYzNWM0F3YVVocFZubHpTVFphWm1wNFpGOHhZMHRzYnpKb1oweGtTRTVETm1WRlJ6Rnphek5FZHpCYWNIRlZTVTEwVmpSV1ZVbDFTVTlxWDFWVk1GWnhSbmh3VVU1WGRuaFFaRTUyZG5jeE0wOUhTMmxVZEZWb2VUZHRhalpDWmxGUFUxQTNWMEp2YmtWc2VIbE1aVmt3V0dGMGJWbElYMlJtUTFsRk56UmhZWE01U3pOUGVEWjFhV0p4VkU4M09IZGtjV2x5VVNKOUxDSmhkV1FpT2lJMmFsTnlRMU01Y210TFdrdEhSblJ3TUVOVlZESWlMQ0psZUhBaU9qRTFPRFU1TVRBek1UTXNJbWxoZENJNk1UVTROVGt3TmpjeE15d2lhWE56SWpvaWFIUjBjSE02THk5c2IyeGpZWFJvYjNOMExtUmxJbjAuQXlBNERIenNGZXJFTDR5a0QydXhENEdXY1FmSDdJY294bEEyNUxqWnFGcnNpQ3J3Mm9GQTB5ZHA0d1l3X3lmTzJNOU9fX2Y0dWFqRHpFVFphcUlrUlhROGw3cEFtMzVna05GVFJBa0ZqTExHS3BLYzI0UkVxTjVBZnBSX2M1NEVkbnBWWlUyNGNYdU5kYzFaZVZyQzJxY19HaEg5STZPVmV5dXFkMnN5X25pXzM3YVNHUFg0WFJhU1RUQ05YOWVyNjhtMWR1RjhmZlRTQ1FUVEFVR09fUm9wcnNGZUQzdHZEWHRLSlBOWVVfYXFtVHFPVnh6bk9FbkpfclN5ZTJaTlF1M0FDcWR2eUtleDl3bmdhV21ULUY2XzQ0RXFqMWwxdW15QlBOSEloVURwcGVqYWpfS2poYVlQZ1ZiRjU1WTZwa002Z0ZUU1NjOVBpMUJscks3QkNRIiwidG9rZW5fdHlwZSI6InBvcCJ9.K21bg2ZaOqt4z8d7FHgo1EazsGlZrNven4CMufcVrECMHcbhKB5YLeJBrXyRL-eVX_PjcehTjVxRXnterwAs-TTpjwrkGGrgJ4pyWanmJK-UqMEj4ezSuPUKqOoYu9cl0z2zi1mUCBU9-gwuj0--IUb6mfiHRXq0YlZqnKDXgkxOMvoh16tHE5Ph-n6_OJD7O4xFzcVaJu-y7x6c7Hi9JFswzqIkT2B0ukcBk95rw690jJhmI02Wd4YPdLAERcv4-CKTJfZFESfdw0CvuJYWjYXta3esrhTMPtBoX2eq8-D4iuGgtIBjFu2Y_58H2Uey-HCpY7xPGqcDATEoceJQHA' -H 'Sec-Fetch-Dest: empty' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36' -H 'Content-Type: application/sparql-update' -H 'Accept: */*' -H 'Origin: https://lolcathost.de' -H 'Sec-Fetch-Site: same-origin' -H 'Sec-Fetch-Mode: cors' -H 'Referer: https://lolcathost.de/' -H 'Accept-Language: nl-NL,nl;q=0.9,en-US;q=0.8,en;q=0.7,fr-FR;q=0.6,fr;q=0.5,de-DE;q=0.4,de;q=0.3,es-ES;q=0.2,es;q=0.1,id-ID;q=0.1,id;q=0.1' -H 'Cookie: _session=Sto-dglZLd_E7_jOw5zoj' --data-binary $' INSERT DATA {<https://lolcathost.de/storage/michiel5/profile/card#me> <https://www.w3.org/TR/activitypub/#following> <https://lolcathost.de/storage/michiel5/friends-fa0ed2d5-ac55-4990-8ab4-af376eee7ffc#this>.\n<https://lolcathost.de/storage/michiel5/friends-fa0ed2d5-ac55-4990-8ab4-af376eee7ffc#this> <http://www.w3.org/ns/ldp#inbox> <https://lolcathost.de/storage/michiel5/friend-requests-inbox/>.\n}' --compressed
```

**Comment by michielbdejong** [comment]
Fixed in release 1.1.0
